http_handler/settings.py

```diff
@@ -83,8 +85,8 @@ def _get_website():
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
-        'NAME': MYSQL["NAME"],# Or path to database file if using sqlite3.
+        'ENGINE': 'django.db.backends.sqlite3', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
```

Comment:
Hi @staciajohanna can you remove the changes made inside DATABASES? Since the hack we had to do for you is not generally how it should be done.

Reply:
fixed